Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Colibri ocolibri = new Colibri();
+            ocolibri.Morir(2022,2023,"hambre");
```

Comment:
Please here use 

 Console.WriteLine(ocolibri.Morir(2022,2023,"hambre");


in order to show the message in the console